# Z Test 

Suppose the IQ in a certain population is normally distributed with a mean of μ = 100 and standard deviation of σ = 15.

A researcher wants to know if a new drug affects IQ levels, so he recruits 20 patients to try it and records their IQ levels.

The following code shows how to perform a one sample z-test in Python to determine if the new drug causes a significant difference in IQ levels.

In [13]:
from statsmodels.stats.weightstats import ztest as ztest

data = [88, 92, 94, 94, 96, 97, 97, 97, 99, 99, 105, 109, 109, 109, 110, 112, 112, 113, 114, 115]

In [15]:
ztest(data,value = 100)

(1.5976240527147705, 0.1101266701438426)

p value = 0.11 which is greater than 0.05(alpha) hence we reject the null hypothesis